app/views/episodes/edit.slim

```diff
@@ -0,0 +1,73 @@
+.container
+  .py-3.text-center
+    img.d-block.mx-auto.mb-4[src="/proconf_logo.jpg" alt="proConf logo" width="72" height="72"]
```

Comment:
Guys! 5 time I write about using CSS
`[src="/proconf_logo.jpg" alt="proConf logo" width="72" height="72"]` move this to class property